lms/static/sass/views/_shoppingcart.scss

```diff
@@ -500,79 +373,89 @@
         }
 
         .col-2 {
-          width: 350px;
-          float: left;
-          line-height: 44px;
+          @include float(left);
+          @include line-height(29.73);
+          margin-top: ($baseline*0.15);
+          width: ($baseline*17.5);
           text-transform: uppercase;
           color: $light-gray2;
-          margin-top: 3px;
 
           .numbers-row {
             position: relative;
-            label{
-              font-size: 16px;
+
+            label {
+              @extend %t-icon5;
+              @extend %t-regular;
               text-transform: uppercase;
               color: $light-gray2;
-              font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-              font-weight: 400;
               font-style: normal;
+              font-family: $sans-serif;
             }
+
             .counter {
-              margin-left: 25px;
-              border-radius: 3px;
-              padding: 6px ($baseline*1.5) 6px ($baseline/2);
+              @include margin-left($baseline*1.25);
+              @include padding(($baseline*0.3), ($baseline*1.5), ($baseline*0.3), ($baseline/2));
```

Comment:
Check: These seem to work just fine, but CSSComb throws an error with the additional parentheses here... Seems to be happy with this though:
`@include padding($baseline*0.3, $baseline*1.5, $baseline*0.3, $baseline/2);`